Holidays update

Vic0303 opened the pull request:
Holidays are now added when opening the program. Redundant files were removed.

Vic0303 committed:
Holidays update

Holidays are now added when opening the program. Redundant files were removed.

Mac-Adam reviewed (changes requested):
Kod nie działa (odpalając maina wyskakuja błędy)
Nawet jażeli by się odpalił, przy każdym otwarciu programu będą się dodawać na nowo te same święta, trzeba zrobić sprawdzenie czy przypadkiem takie święto nie jest już dodane

Vic0303 committed:
Update ourCalendar.py

Vic0303 committed:
Update ourCalendar.py

Modified load_from_file so that holidays are not duplicated. Added static method is_polish_holiday used in the previous function.
